maas/resource_maas_instance.go

```diff
@@ -197,6 +198,41 @@ func resourceMAASInstance() *schema.Resource {
 				Computed:    true,
 				Description: "The deployed MAAS machine pool name.",
 			},
+			"release_params": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "Parameters used to release the allocated machine when the resource is destroyed.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"comment": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "A comment to be added to the event log when the machine is released.",
+						},
+						"erase": {
+							Type:        schema.TypeBool,
+							Optional:    true,
+							Description: "Erase the disk when releasing.",
+						},
+						"force": {
```

Comment:
I know the MAAS API has this as force, but i'm wondering whether `force_release` or something slightly more informative might be useful for the terraform side @skatsaounis ?

Reply:
I have no strong preference here. Typically, I follow the API so that users that using both are not confused.